test/jdk/java/awt/Robot/ModifierRobotKey/ModifierRobotEnhancedKeyTest.java

```diff
@@ -155,7 +154,7 @@ private void initializeGUI() {
     }
 
     public void doTest() throws Exception {
-        robot = new ExtendedRobot();
```

Comment:
`robot.click()` at line 161 is from ExtendedRobot class. Can u reconfirm that ExtendedRobot is not required here and other classes?